release-blocking-jobs.md

```diff
@@ -87,9 +87,31 @@ manual, human interpretation, we choose to move them to a separate dashboard.
 
 CI Signal Team members will regularly check the Informing Dashboard for new
 failures.  Unexplained failures that are not flakes should be considered reason
-to block the release until the failures are investigated and evaluated.  Unlike
-Blocking, however, jobs in Informing do not need to be green before authorizing
-a release, they can be failing for known and "acceptable" reasons.
+to block the release until the failures are investigated and evaluated.
+Unlike Blocking, however, jobs in Informing do not need to be green before
+authorizing a release, they can be failing for known and "acceptable" reasons.
+
+The criteria for deciding whether a failure in informing is to be ignored can be
+based on the following considerations:
+
+* Failure is due to a malfunction of the components that run a job and said
+  components are not related to Kubernetes
+  * For example, if a job is marked as failing because a test cluster could not be setup
+    but there are similar jobs that run the same tests, then the failing job
+    can be ignored.
+  * If a job's stage, a stage that is not related to any Kubernetes e2e test, fails
+    (e.g., couldn't upload some logs to a remote source) then the job failure can be
+    ignored given that the stage doesn't disrupt any e2e tests.
+  * If a test-infra-related component fails and no there are no other factors
+    that could be responsible for the failure then the failure can be ignored.
+* Failure well understood and is caused by the configuration of a test and said
+  configuration is not related to an official SLO
+  * For example, if a test polls for metrics more fequently than these are
+    updated and fails because of it, then the failure may be ignored.
+* Job is flaking too frequently and is marked as failing
+  * This should only be done when different tests are flaking. If the same set
+    of tests are flaking then we should fix them.
+* A test failed and was subsequently removed, thus marking the job as failed.
```

Comment:
meaning "the last info we have on this test is that its status is failed but it is irrelevant now?"

Reply:
Yes. This is completely empirical and something that I know has been done in the past.
Because e2e tests are chosen based on tags that are specified in jobs (i.e., jobs run "all tests that have label X as opposed to running a prescribed list of tests), it is possible for someone to include a e2e test in a job by accident, or to include it during development and then remove it, among some other instances.

Because of these reasons, in the past there have been instances where a test appears, it fails, then it is removed. Tesgrid is only aware that that e2e test has not had a successful run so it can mark a job that is otherwise healthy as failing until it forgets about the test that disappeared (which can take multiple days).